backend/app/api/v1/endpoints/quay/quayJobs.py

```diff
@@ -82,3 +82,54 @@ async def jobs(
 
     jsonstring = json.dumps(response)
     return jsonstring
+
+
+@router.get(
+    "/api/v1/quay/filters",
+    summary="Returns a filter list",
+    description="Returns the data to build filters in the specified dates. \
+            If not dates are provided the API will default the values. \
+            `startDate`: will be set to the day of the request minus 5 days.\
+            `endDate`: will be set to the day of the request.",
```

Comment:
Are the last two lines the default values? If so, I think a few words explicitly mentioning that those are the default values would be good. Maybe wording like "the API will use the following default values:" or "the API will use the following values as defaults:"

Reply:
Noted!